### PR TITLE
SAIC-27: Quantum support

### PR DESCRIPTION
**Pull request by miarmak**
- Test networks migration from Grizzly (Quantum) to Icehouse (Neutron).
   Result: Networks migrated successfully
- Fix README.md
- Delete obsolete TODO's about Quantum

### CODE REVIEW

**Comment by Wasapl** [comment]
Merged build finished. Test PASSed.  http://jenkins-cf.vm.mirantis.net:8080/job/CloudFerry_PR_checks/100/

**Comment by mirrorcoder** [comment]
+1

File: configs/config.ini [hunk 1]
```diff
@@ -16,6 +16,7 @@ direct_compute_transfer=yes
 keep_lbaas = no
 ssh_chunk_size = 100
 retry = 5
+#migrate_extnets = True
```
Comment (oleksii-shyman) [inline configs/config.ini]: why this is commented?

**Comment by miarmak** [comment]
Fixed

**Comment by Wasapl** [comment]
Merged build finished. Test PASSed.  http://jenkins-cf.vm.mirantis.net:8080/job/CloudFerry_PR_checks/115/

File: README.md [hunk 2]
```diff
@@ -52,18 +52,21 @@ to implement in nearest future.
...
 - ipaddr
+- jinja2
+- mysql-connector-python
+- pycrypto
+- python-cinderclient==1.0.9
+- python-glanceclient==0.12.0
+- python-keystoneclient==0.9.0
+- python-neutronclient==2.3.4
+- python-novaclient==2.15.0
+- python-swiftclient==2.3.1
```
Comment (roman-verchikov) [inline README.md]: I wonder why we even need the list of dependencies here, it simply duplicates `requirements.txt`, meaning we'd have to maintain both `README.md` and `requirements.txt` synchronized (the downsides seem obvious). For this patchset: 

1. Let's only leave relevant modification (the one with `neutron` changed to `quantum/neutron`)
2. The rest of modifications to README.md should go in a separate patch. In that one we should simply remove all the python library requirements and simply replace it with a phrase "all the python requirements are listed in requirements.txt".

**Comment by roman-verchikov** [comment]
Please rebase and fix inline

**Comment by Wasapl** [comment]
Build finished. Test PASSed.  http://jenkins-cf.vm.mirantis.net:8080/job/CloudFerry_PR_checks/132/

**Comment by roman-verchikov** [comment]
Looks like this still needs rebase

**Comment by miarmak** [comment]
@roman-verchikov yep, waiting for decision here: pull #125

**Comment by miarmak** [comment]
Rebased

**Comment by Wasapl** [comment]
Merged build finished. Test FAILed.   http://jenkins-cf.vm.mirantis.net:8080/job/CloudFerry_PR_checks/136/

**Comment by roman-verchikov** [comment]
This needs one more rebase

**Comment by miarmak** [comment]
Rebased

**Comment by Wasapl** [comment]
Merged build finished. Test FAILed.   http://jenkins-cf.vm.mirantis.net:8080/job/CloudFerry_PR_checks/150/

**Pull request merged**